### DISCUSSION TITLE
Optimize for Android tablets

### DISCUSSION THREAD

**Discussion by cbrnr** [community/community#3476] [Mobile]
While I'm enjoying GitHub on my Android phone a lot, the experience on an Android tablet is suboptimal - it's literally the app for mobile phones blown up to larger screens. It would be super useful if the extra screen estate on tablets could be used to show additional things instead of just containing a lot of empty space.

Since this experience is already available on iPad, I'd love to see the same layout supported on Android tablets. Are there any plans on implementing Android tablet support in the near future?

**Comment by brianlovin** [comment]
Thanks for the feedback @cbrnr! I've added a +1 to our internal tracking issue for better Android tablet support. Right now this isn't planned or on our roadmap for the next few months, but feedback like this from people helps us prioritize down the road 😊
